### ISSUE TITLE
Italicize foriegn quotes

### ISSUE THREAD

**Issue by thewchan** [acreese/willa-cather_o-pioneers#3]
Some foreign quotes and passage have the correct `<span xml:lang= ...` tags, but they need to be italicized even if they are already in quotations. Here are some examples:

chapter-2-4 line 22:
>`<span epub:type="z3998:song" xml:lang="de">‘Wo bist du, wo bist du, mein geliebtest Land?’</span>`

chapter-4-1 line 28:
>`<p><span xml:lang="fr">“Mais, oui,”</span> said the priest, with a melancholy smile. <span xml:lang="fr">“C’est L’Isle-Adam, chez ma mère. Vous êtes très savante, ma fille.”</span> He patted her yellow turban, calling, <span xml:lang="fr">“Venez donc, mes garçons! Il y a ici une veritable clairvoyante!”</span></p>`

There may be others, but those are the ones I've found

**Comment by acreese** [comment]
Thanks for taking a look at this, Matt! I'll get to work on this today. I
didn't italicize these because they weren't italic in the scan of the book,
but based on our manual of style I should italicize anyway?

**Comment by thewchan** [comment]
Yup; check out this section of the manual: https://standardebooks.org/manual/1.6.1/single-page#8.2.9

Alex clarified in the email thread that the quotation marks should not be in the `<i>` tags
